.travis.yml

```diff
@@ -97,42 +107,63 @@ install:
 script:
 - set -e
 - if [[ "${BUILD_TARGET}" = "gcc_build" ]]; then
-    make BUILD_TYPE=Debug run_unit_tests -j4;
-    make clean;
-    make INSTALL_PREFIX=/usr BUILD_TYPE=Release run_unit_tests -j4;
+    cmake -DCMAKE_BUILD_TYPE=Debug -Bbuild/debug -H.;
+    make -Cbuild/debug -j4;
```

Comment:
I would probably use `(cd build/debug && cmake --build .)`. That way it can use either Unix Makefiles or ninja.

Reply:
No need to change dir, btw: `cmake --build build/debug`.

Reply:
And you're telling me now! I could have used that 5 years ago :smile: